packages/itwin/tree-widget/src/components/trees/models-tree/HierarchyBasedVisibilityHandler.ts

```diff
@@ -733,8 +752,8 @@ class VisibilityHandlerImplementation implements IVisibilityHandler {
     );
   }
 
-  private getAlwaysOrNeverDrawnChildren(props: ElementsQueryProps & { always: boolean }): Observable<Id64Set | undefined> {
-    const set = props.always ? this._props.viewport.alwaysDrawn : this._props.viewport.neverDrawn;
+  private getAlwaysOrNeverDrawnChildren(props: ElementsQueryProps & { setType: "always" | "never" }): Observable<Id64Set | undefined> {
```

Comment:
why are all these functions called `get*Children`? any specific reason to use `Children` instead of `Elements`?

Reply:
Changed it so that these names now refer to `elements`, `children` (as element children) or `treeNodeChildren` (as category, model or grouping node elements)